amazon_msk/datadog_checks/amazon_msk/data/conf.yaml.example

```diff
@@ -90,10 +90,14 @@ instances:
 
     ## @param label_joins - mapping - optional
     ## Allows targeting a metric to retrieve its label with a 1:1 mapping.
+    ## Putting multiple values in `labels_to_match` works as an `AND` operator (all labels must match)
+    ## A ['*'] value in `labels_to_match` allows to apply retrieved labels to all timeseries (match all)
+    ## A ['*'] value in `labels_to_get` allows to retrieve all labels from timeserie
     #
     # label_joins:
     #   target_metric:
-    #     label_to_match: <MATCHED_LABEL>
+    #     labels_to_match:
```

Comment:
Is this change of name a breaking change?

Reply:
No, we still support `label_to_match`. Currently `label_to_match` is deprecated and it produces a warning log each time it's used. Feature was introduced few months ago but documentation was not updated.